Core/DLC/BreakingGroundDlcDetector.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace CKAN.DLC
+{
+    /// <summary>
+    /// Represents an object that can detect the presence of the official Making History DLC in a KSP installation.
+    /// </summary>
+    public sealed class BreakingGroundDlcDetector : StandardDlcDetectorBase
+    {
+        public BreakingGroundDlcDetector()
+            : base("BreakingGround", new Dictionary<string, string>()
```

Comment:
Apparently the folder name for this DLC is `Serenity`, so looking for a `BreakingGround` folder doesn't work. And we probably don't want to make modders depend on `Serenity-DLC`, so can we split out the identifier from the folder?

Reply:
Well that's... interesting. I'll have an update tonight once I'm home.